{
  "type": "issue",
  "issue": {
    "id": 125053511,
    "node_id": "MDU6SXNzdWUxMjUwNTM1MTE=",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/7301",
    "number": 7301,
    "state": "closed",
    "state_reason": "completed",
    "title": "syncing stopped. coinbase pays too much",
    "body": "I run the latest bitcoin core (v0.11.2 64 bit) on my debian and syncing stopped at height 389844. I tried to do what GMaxwell suggested in #6840 : I ran reconsiderblock with that and the follow up block according to [some block explorer](https://www.blocktrail.com/BTC/block/0000000000000000068234693e41decd5bfc1b8c5d62cc0aee8a78b398d39cb8):\n\n> > reconsiderblock 0000000000000000068234693e41decd5bfc1b8c5d62cc0aee8a78b398d39cb8\n> > reconsiderblock 00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e\n\nand in the debug console it gave me no feedback but in the log I find how these are being considered and rejected with **coinbase pays too much (actual=2526435001 vs limit=2526435000)**\n\nHow do I recover? How can I help you prevent this from happening generally?\n\n```\n2016-01-05 19:10:23 ERROR: ConnectTip(): ConnectBlock 00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e failed\n2016-01-05 19:10:23 InvalidChainFound: invalid block=00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e  height=389845  log2_work=83.788191  date=2015-12-23 16:43:10\n2016-01-05 19:10:23 InvalidChainFound:  current best=0000000000000000068234693e41decd5bfc1b8c5d62cc0aee8a78b398d39cb8  height=389844  log2_work=83.788156  date=2015-12-23 16:37:12\n2016-01-05 19:11:01 ERROR: AcceptToMemoryPool: nonstandard transaction: dust\n2016-01-05 19:11:46 ERROR: ConnectBlock(): **coinbase pays too much** (actual=2526435001 vs limit=2526435000)\n2016-01-05 19:11:51 InvalidChainFound: invalid block=00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e  height=389845  log2_work=83.788191  date=2015-12-23 16:43:10\n2016-01-05 19:11:51 InvalidChainFound:  current best=0000000000000000068234693e41decd5bfc1b8c5d62cc0aee8a78b398d39cb8  height=389844  log2_work=83.788156  date=2015-12-23 16:37:12\n2016-01-05 19:11:51 ERROR: ConnectTip(): ConnectBlock 00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e failed\n2016-01-05 19:11:51 InvalidChainFound: invalid block=00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e  height=389845  log2_work=83.788191  date=2015-12-23 16:43:10\n2016-01-05 19:11:51 InvalidChainFound:  current best=0000000000000000068234693e41decd5bfc1b8c5d62cc0aee8a78b398d39cb8  height=389844  \n```\n",
    "user": {
      "login": "Giszmo",
      "id": 96378,
      "node_id": "MDQ6VXNlcjk2Mzc4",
      "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/Giszmo",
      "html_url": "https://github.com/Giszmo",
      "followers_url": "https://api.github.com/users/Giszmo/followers",
      "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
      "organizations_url": "https://api.github.com/users/Giszmo/orgs",
      "repos_url": "https://api.github.com/users/Giszmo/repos",
      "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/Giszmo/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": true,
    "active_lock_reason": "resolved",
    "comments": 6,
    "closed_at": "2016-01-05T21:24:17Z",
    "created_at": "2016-01-05T21:17:35Z",
    "updated_at": "2021-09-08T12:53:33Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 169137667,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTEzNzY2Nw==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169137667",
      "actor": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:21:56Z",
      "updated_at": "2016-01-05T21:21:56Z",
      "author_association": "CONTRIBUTOR",
      "body": "That likely means that your UTXO set database is corrupt.\n\nTypically that's the result of a hardware failure with your hard drive.\n\nYou will need to run with the -reindex flag set.\n",
      "user": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169137667",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "commented",
      "id": 169138220,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTEzODIyMA==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169138220",
      "actor": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:24:17Z",
      "updated_at": "2016-01-05T21:28:26Z",
      "author_association": "NONE",
      "body": "@pstratem thank you for your lightning fast reply. Interesting. Will check the disk, too.\n\nThe machine is totally not up to the task and on several occasions went into emergency shutdown due to overheating. Which of course also increases chances for disc failure a bit but for now I'll just check things out and assume this was it. So the UTXO set has no integrity check yet? There's talks of putting the UTXO-set hash into the block header, right?\n",
      "user": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169138220",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "mentioned",
      "id": 506408721,
      "node_id": "MDE0Ok1lbnRpb25lZEV2ZW50NTA2NDA4NzIx",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/506408721",
      "actor": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:24:17Z"
    },
    {
      "event": "subscribed",
      "id": 506408722,
      "node_id": "MDE1OlN1YnNjcmliZWRFdmVudDUwNjQwODcyMg==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/506408722",
      "actor": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:24:17Z"
    },
    {
      "event": "closed",
      "id": 506408723,
      "node_id": "MDExOkNsb3NlZEV2ZW50NTA2NDA4NzIz",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/506408723",
      "actor": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:24:17Z"
    },
    {
      "event": "commented",
      "id": 169141485,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTE0MTQ4NQ==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169141485",
      "actor": {
        "login": "paveljanik",
        "id": 6848764,
        "node_id": "MDQ6VXNlcjY4NDg3NjQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6848764?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/paveljanik",
        "html_url": "https://github.com/paveljanik",
        "followers_url": "https://api.github.com/users/paveljanik/followers",
        "following_url": "https://api.github.com/users/paveljanik/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/paveljanik/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/paveljanik/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/paveljanik/subscriptions",
        "organizations_url": "https://api.github.com/users/paveljanik/orgs",
        "repos_url": "https://api.github.com/users/paveljanik/repos",
        "events_url": "https://api.github.com/users/paveljanik/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/paveljanik/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:39:05Z",
      "updated_at": "2016-01-05T21:39:05Z",
      "author_association": "CONTRIBUTOR",
      "body": "Can you show us the rawblock 00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e? E.g.\n\n```\n$ bitcoin-cli getblock 00000000000000000842286faa1f37fd609e1374ada25c0aa9c60155cc32425e false | md5sum \n20a87cbae0f381ebd1adab3de7607191  -\n\n```\n",
      "user": {
        "login": "paveljanik",
        "id": 6848764,
        "node_id": "MDQ6VXNlcjY4NDg3NjQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6848764?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/paveljanik",
        "html_url": "https://github.com/paveljanik",
        "followers_url": "https://api.github.com/users/paveljanik/followers",
        "following_url": "https://api.github.com/users/paveljanik/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/paveljanik/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/paveljanik/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/paveljanik/subscriptions",
        "organizations_url": "https://api.github.com/users/paveljanik/orgs",
        "repos_url": "https://api.github.com/users/paveljanik/repos",
        "events_url": "https://api.github.com/users/paveljanik/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/paveljanik/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169141485",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "commented",
      "id": 169143390,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTE0MzM5MA==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169143390",
      "actor": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-05T21:46:32Z",
      "updated_at": "2016-01-05T21:46:32Z",
      "author_association": "NONE",
      "body": ":/ sorry, it's already re-indexing. Gives me an error: \"Block not found\" in the debug console and with bitcoin-cli \"couldn't connect to server\".\n",
      "user": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169143390",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "commented",
      "id": 169534374,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTUzNDM3NA==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169534374",
      "actor": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-07T03:02:03Z",
      "updated_at": "2016-01-07T03:02:03Z",
      "author_association": "NONE",
      "body": "`badblocks` and `touch /forcefsck` did not yield any indications of a bad disk. The ram check found one bad bit in the 8GB in one of 8 runs. Also restarting bitcoin gave me a bad merkel root or so. Three lines of error, 60 times per second, preventing a normal shut down of bitcoin. Only `kill -9` did the trick.\n",
      "user": {
        "login": "Giszmo",
        "id": 96378,
        "node_id": "MDQ6VXNlcjk2Mzc4",
        "avatar_url": "https://avatars.githubusercontent.com/u/96378?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Giszmo",
        "html_url": "https://github.com/Giszmo",
        "followers_url": "https://api.github.com/users/Giszmo/followers",
        "following_url": "https://api.github.com/users/Giszmo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Giszmo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Giszmo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Giszmo/subscriptions",
        "organizations_url": "https://api.github.com/users/Giszmo/orgs",
        "repos_url": "https://api.github.com/users/Giszmo/repos",
        "events_url": "https://api.github.com/users/Giszmo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Giszmo/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169534374",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "commented",
      "id": 169534619,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTUzNDYxOQ==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/169534619",
      "actor": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2016-01-07T03:04:17Z",
      "updated_at": "2016-01-07T03:04:17Z",
      "author_association": "CONTRIBUTOR",
      "body": "> The ram check found one bad bit in the 8GB in one of 8 runs.\n\nDuring initial block synchronization your computer is being pushed to it's limits.\n\nIf you found memory errors under zero load then there's going to be LOTS of errors under thermal load.\n",
      "user": {
        "login": "pstratem",
        "id": 620611,
        "node_id": "MDQ6VXNlcjYyMDYxMQ==",
        "avatar_url": "https://avatars.githubusercontent.com/u/620611?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/pstratem",
        "html_url": "https://github.com/pstratem",
        "followers_url": "https://api.github.com/users/pstratem/followers",
        "following_url": "https://api.github.com/users/pstratem/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/pstratem/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/pstratem/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/pstratem/subscriptions",
        "organizations_url": "https://api.github.com/users/pstratem/orgs",
        "repos_url": "https://api.github.com/users/pstratem/repos",
        "events_url": "https://api.github.com/users/pstratem/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/pstratem/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/7301#issuecomment-169534619",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/7301"
    },
    {
      "event": "locked",
      "id": 5272118862,
      "node_id": "LOE_lADOABII584HdCpHzwAAAAE6PiZO",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/5272118862",
      "actor": {
        "login": "MarcoFalke",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/MarcoFalke",
        "html_url": "https://github.com/MarcoFalke",
        "followers_url": "https://api.github.com/users/MarcoFalke/followers",
        "following_url": "https://api.github.com/users/MarcoFalke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/MarcoFalke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/MarcoFalke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/MarcoFalke/subscriptions",
        "organizations_url": "https://api.github.com/users/MarcoFalke/orgs",
        "repos_url": "https://api.github.com/users/MarcoFalke/repos",
        "events_url": "https://api.github.com/users/MarcoFalke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/MarcoFalke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2021-09-08T12:53:33Z",
      "lock_reason": "resolved"
    }
  ]
}